plugins/org.eclipse.osee.ats.core/src/org/eclipse/osee/ats/core/action/AtsActionService.java

```diff
@@ -545,9 +544,10 @@ public IAtsTeamWorkflow createTeamWorkflow(IAtsAction action, IAtsTeamDefinition
             artToken = listener.getArtifactToken(applicableAis);
          }
       }
-
+      
       if (artToken == null) {
-         teamWf = atsApi.getWorkItemService().getTeamWf(changes.createArtifact(artifactType, ""));
+         String name = (action.getName() == null) ? Strings.EMPTY_STRING : action.getName();
```

Comment:
Question...is there a sensible default we could use instead? Like TW(NEXT_NUMBER_HERE)?